public/static/plugins/buildbaron/js/task_build_baron.js

```diff
@@ -28,6 +28,12 @@ mciModule.controller('TaskBuildBaronCtrl', function($scope, $http, $window) {
             }
             // next, prefer tickets resolved with "Fixed"
             if (a.statusKey == 2) {
+              // The alternative suggestion endpoint may not return the resolution of tickets, even
+              // if their status is "Fixed" or "Closed". We fall back to comparing the tickets by
+              // their updated date.
+              a.fields.resolution = a.fields.resolution || {};
```

Comment:
is `*.fields.resolution` an object? It appears to be a date, but i'm not sure if that means date object or string. Either way, null is probably more appropriate here.

Reply:
It is meant to be an object with a "name" property. If we made it the literal `null` value, then we'd trigger a `TypeError` and would fail to render the data from the alternative endpoint in the Build Baron plugin.

Reply:
Got it. Sounds good!